app/client/components/helpCentre/helpCentre.tsx

```diff
@@ -86,6 +87,13 @@ export const HelpCentre = (_: HelpCentreProps) => (
           fontWeight={"bold"}
           text="Take me to the form"
           to="/contact-us/"
+          onClick={() =>
+            trackEvent({
+              eventCategory: "help-centre",
+              eventAction: "cta-click",
+              eventLabel: "contact-us-form"
```

Comment:
To be more inline with the other events we're tracking I suggest that the `eventAction` is set to `contact-us-cta-click` and that we remove the `eventLabel`. What do you think?